pyproject.toml

```diff
@@ -31,6 +31,8 @@ dependencies = [
   "bids-validator",
   "num2words",
   "click >=8.0",
+  "plotly",
+  "ipywidgets >=7.0.0"
```

Comment:
We really should be careful about mandatory requirements. With this change every BIDS App that uses pybids will need to install plotly and ipywidgets, regardless of whether they use this functionality.

Reply:
yup that's why I am considering that this whole thing may better be a pybids extension, no?

Reply:
That would certainly simplify things from my perspective.

Reply:
ok let's make our life easier and do this.

are you ok finishing the review here and then I just transfer the whole thing to a separate repo?

Reply:
I was going to say the same thing. I think this would be a good extension. 

The plots looks nice! However, it will probably remain somewhat niche so that's a better format for this.

In general we're moving towards breaking pybids apart (or we should be at least).